src/components/send/components/Breakdown.css.js

```diff
@@ -0,0 +1,29 @@
+import styled from 'styled-components';
```

Comment:
Not a blocker, but something to consider:

If we're only going to use this with `AvailableToSend.js`, it'd be nice to get `AvailableToSend` into a directory with this css file co-located with it, especially since there's nothing about the filename to indicate that it is part of (and only part of) the `AvailableToSend` component...

If we do intend to re-use it in multiple components, I wonder if it belongs in an e.g. `css` or similarly named directory rather than co-located with actual react components for clarity that this is a styled div, but not an actual react component...